cockroachcloud/cmek.md

```diff
@@ -108,6 +108,7 @@ Backups in {{ site.data.products.dedicated }} are triggered in two ways, only on
 ## Limitations
 The CMEK feature has the following limitations:
 
+- During limited access, CMEK is not available for {{ site.data.products.dedicated }} clusters on Azure. Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
 - CMEK can be enabled only on clusters created after April 1, 2022 (AWS) or June 9, 2022 (GCP).
 - The cluster must be a private cluster. Refer to [Create Private Clusters](private-clusters.html)
```

Comment:
I think we should change it to indicate that CMEK can be enabled only on Dedicated Advanced plan, which means private clusters by default.